### PR TITLE
[Hall of Fame]2 - Card template

### PR DESCRIPTION
**Pull request by NosatskyiK**

### CODE REVIEW

**Comment by IgorFromUA** [comment]
Hello. You can customize the container to align the content.

**Comment by NosatskyiK** [comment]
> Hello. You can customize the container to align the content.  

Hello, Bohdan worked with the text, so I think it would be correct to write to him about the text on the page

**Review by Bogdandrummer — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -2,67 +2,144 @@
...
+                                </div>
+                            </a>
+                            <a href="#" data-telegram class="social__link">
+                                <div class="card__social-telegram">
+                                    <div class="social__img-telegram">
+                                    </div>
+                                </div>
+                            </a>
+                        </div>
+                        <div class="card__btn">
```
[the remark this answers is not in the document]
Comment (NosatskyiK) [reply docs/index.html]: removed unused tag classes

**Comment by vladyslav-yermolin-moc** [comment]
Hey, guys, I've added some minor styles corrections, and going to make a code-review

**Comment by vladyslav-yermolin-moc** [comment]
The icons in the footer are blocked by CORS 
image

**Comment by vladyslav-yermolin-moc** [comment]
Ok, the Github pages don't like the underscores in the file name (probably).
So I've changed the files structure and corrected the pathes
Here is deployed version (on my Github pages) https://vladyslav-yermolin-moc.github.io/fe-course-basic-2023/

File: docs/src/styles/null.css
Comment (vladyslav-yermolin-moc) [inline docs/src/styles/null.css:1]: What a name!!!)))
Please use more common: reset or normalize.css

File: docs/src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  max-width: 930px;
+  margin: 0 auto;
+}
+
+.main {
+  flex-grow: 1;
+}
+
+.main__title {
+  font-family: Roboto sans-serif;
```
Comment (vladyslav-yermolin-moc) [inline docs/src/styles/style.css:27]: Something wrong with font connection - please don't forget to add safe font and tune up connection of Roboto properly

File: docs/src/styles/styles.css
Comment (vladyslav-yermolin-moc) [inline docs/src/styles/styles.css:1]: Probably, the `card.css` would be more suitable name

File: docs/src/styles/style.css
Comment (vladyslav-yermolin-moc) [inline docs/src/styles/style.css:1]: Please add some suitable name for this file

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
I'm going to merge this pr, but fixes of my comments here are on you, @NosatskyiK

**Pull request merged**